### PR TITLE
Implement message headers attribute

### PR DESCRIPTION
**Pull request by ducpt85**
Implement the `Message.headers` attribute.

### CODE REVIEW

**Comment by ducpt85** [comment]
@wenxiang @giantballofyarn please help to review and merge if it's good

File: src/kafka_cffi/message.py [hunk 1]
```diff
@@ -24,6 +25,25 @@ def __init__(self, rkmessage):
...
+        c_hd_payload = ffi.new("const char **")
+        c_hd_size = ffi.new("size_t *")
+        self.__headers = []
+        for idx in range(hdcount):
+            error = lib.rd_kafka_header_get_all(
+                c_header, idx, c_hd_key, ffi.cast("const void **", c_hd_payload), c_hd_size)
+            if error == 0:
+                self.__headers.append(
+                    (ffi.string(c_hd_key[0]),
+                     ffi.string(c_hd_payload[0], int(c_hd_size[0])))
```
Comment (hyperair) [inline src/kafka_cffi/message.py]: This is potentially binary data, right? Should we be using `ffi.unpack` instead of `ffi.string` here?

Comment (ducpt85) [reply src/kafka_cffi/message.py]: You are right. I need to do more checking on the type of header value before casting I guess

Comment (hyperair) [reply src/kafka_cffi/message.py]: If I'm understanding this correctly, the header value is binary data, so perhaps we should just leave it as binary data (`bytes` in python3, as returned by `ffi.unpack`), and let the API user handle it?

Comment (ducpt85) [reply src/kafka_cffi/message.py]: @hyperair 💇‍♂

**Pull request merged**